explorer/explorertypes.go

```diff
@@ -334,6 +357,18 @@ type BlockSubsidy struct {
 	Dev   int64 `json:"dev"`
 }
 
+// MempoolData models data to update mempool info on the home page
```

Comment:
(on the new home page).  Reading the same doc as `MempoolInfo` is a bit confusing.